tools/llm_bench/llm_bench_utils/ov_utils.py

```diff
@@ -218,22 +233,15 @@ def create_genai_text_gen_model(model_path, device, ov_config, **kwargs):
     cb = kwargs.get("use_cb", False)
     if cb or draft_model_path:
         log.info("Continuous Batching mode activated")
-        default_cb_config = {"cache_size": 1}
-        scheduler_config = openvino_genai.SchedulerConfig()
-        scheduler_params = kwargs.get("cb_config") or default_cb_config
-        if scheduler_params:
-            log.info(f"Scheduler parameters:\n{scheduler_params}")
-
-            for param, value in scheduler_params.items():
-                setattr(scheduler_config, param, value)
-        ov_config["scheduler_config"] = scheduler_config
+        ov_config["scheduler_config"] = get_scheduler_config_genai("cb_config", **kwargs)
 
     if draft_model_path:
         if not Path(draft_model_path).exists():
             raise RuntimeError(f'==Failure ==: draft model by path:{draft_model_path} is not exists')
         log.info("Speculative Decoding is activated")
-
-        ov_config['draft_model'] = openvino_genai.draft_model(draft_model_path, kwargs['draft_device'].upper())
+        draft_device = kwargs.get('draft_device', None) or device
+        ov_config['draft_model'] = openvino_genai.draft_model(draft_model_path, draft_device.upper(),
+                                                              scheduler_config=get_scheduler_config_genai("draft_cb_config", **kwargs))
```

Comment:
if draft model cb config is not provided, then it should be not set or used the same like for target model

Reply:
also you need to add parcing for this config file similar to original cb config https://github.com/openvinotoolkit/openvino.genai/blob/master/tools/llm_bench/llm_bench_utils/model_utils.py#L142

Reply:
updated